mmeval/metrics/matting_mse.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import numpy as np
+from typing import Dict, List, Sequence
+
+from mmeval.core import BaseMetric
+
+
+class MattingMSE(BaseMetric):
+    """Mean Squared Error metric for image matting.
+
+    This metric compute per-pixel squared error average across all
+    pixels.
+    i.e. mean((a-b)^2) / norm_const
+
+    .. note::
+
+        Current implementation assume image / alpha / trimap array in numpy
+        format and with pixel value ranging from 0 to 255.
+
+    .. note::
+
+        pred_alpha should be masked by trimap before passing
+        into this metric
+
+    Default prefix: ''
```

Comment:
This can be removed~

Reply:
Thank you for reminding me that it has been deleted in the latest submission.